supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesAsyncBackendJobExecutionActor.scala

```diff
@@ -238,15 +264,29 @@ class TesAsyncBackendJobExecutionActor(override val standardParams: StandardAsyn
     }
   }
 
+  private def addSystemLogsToStatusAsync(status: TesRunStatus, handle: StandardAsyncPendingExecutionHandle): Future[TesRunStatus] = {
+    makeRequest[Task](HttpRequest(uri = s"$tesEndpoint/${handle.pendingJob.jobId}?view=FULL")) map { response =>
+      status.setStatus(response.state.getOrElse(status.status))
+      status.setLogs(response.logs.last(0).system_logs.getOrElse(Seq.empty[String]))
+      status
+    }
+  }
+
   override def customPollStatusFailure: PartialFunction[(ExecutionHandle, Exception), ExecutionHandle] = {
     case (oldHandle: StandardAsyncPendingExecutionHandle@unchecked, e: Exception) =>
       jobLogger.error(s"$tag TES Job ${oldHandle.pendingJob.jobId} has not been found, failing call")
       FailedNonRetryableExecutionHandle(e, kvPairsToSave = None)
   }
 
+  private def handleExecutionError(status: TesRunStatus, returnCode: Option[Int]): Future[ExecutionHandle] = {
+    val exception = new AggregatedMessageException(s"Task ${jobDescriptor.key.tag} failed for unknown reason: ${status.status}", status.sysLogs)
```

Comment:
Do we expect `sysLogs` to be a single error message, or is it the full system logs for the task? Should we limit the amount of it that we log?

Reply:
In the CoA implementation of TES, there will usually be an even number of items, typically two if any, although sometimes more, but in **almost** every case I've ever seen, never more than 4. Typically, one of those (it's never clear which one) will contain the best clue as to what went wrong.